docs/components/Combobox/Combobox.stories.mdx

```diff
@@ -25,22 +25,50 @@ adding a new item to the list of options or managing a selection.
 
 Combobox is composed of three main parts:
 
-#### Trigger
+#### Activator
 
-The trigger is the user-facing interface to open and close the Combobox and
-initiate option selection. The choice between them depends on the design and
-user experience requirements.
+The activator is the user-facing interface to open and close the Combobox and
+initiate option selection. When implementing Combobox, you may rely on the
+default activator or, you may pass in a custom activator.
 
-There are two options for trigger elements of the Combobox,
-[TriggerButton](../?path=/story/components-selections-combobox-web--trigger-button)
+##### Default activator
+
+The default activator is a
+[Chip](../?path=/story/components-selections-chip-web--subtle) containing an
+`add` Icon suffix and an optional Combobox prop `label` that by default, reads
+"Select". The label can be customized by adding the `label` prop to Combobox, to
+better reflect the type of data being selected. Examples of using a default
+activator can be found in the
+[single-select](../?path=/code/components-selections-combobox-web--single-select)
 and
-[TriggerChip](../?path=/story/components-selections-combobox-web--trigger-chip).
-You must have one trigger component.
+[multi-select](../?path=/code/components-selections-combobox-web--multi-select)
+stories.
+
+_Note:_
+
+- The optional `label` prop on Combobox will only affect the default activator.
+  A custom activator label will overwrite the Combobox label.
+- Omitting a custom activator subcomponent will result in the default activator
+  being used.
+
+##### Custom activator
+
+The custom activator should be used when the Combobox activator is a
```

Comment:
Similar to above, I know we have some actual restrictions on this so we might have to be a bit more verbose, but can we embed an example so we can do more showing, less telling?